package.json

```diff
@@ -7,7 +7,7 @@
     "test": "mocha"
   },
   "bin": {
-    "repo-list": "cli.js"
+    "mongodb-repo-list": "cli.js"
```

Comment:
i spoke briefly with @kangas, and he suggested that the bin name should be the same as the module name, which we are keeping at mongodb-repo-list to stay consistent with the convention of other modules in mongodb-js